pages/index.html

```diff
@@ -10,7 +10,7 @@
     <li>
       <a href="https://twitter.com/share" class="twitter-share-button"
          data-url="http://www.mixxx.org"
-         data-text="{% blocktrans %}Check out Mixxx - Free MP3 DJ mixing software for Windows, Mac OS X, and Linux{% endblocktrans %}"
+         data-text="{% blocktrans %}Check out Mixxx - Free Digital DJ MP3 mixing software for Windows, Mac OS X, and Linux{% endblocktrans %}"
```

Comment:
I'm not too worried about being careful about the meaning of "free" in the title text of the web page. I think the rest of the page makes it pretty clear. I agree that putting "Free & open source" could help.

I'd like to say just "Digital DJ mixing software", but I think the "MP3" is there for search engine optimization. Mixxx.org appears as the 4th Google search result for "mp3 dj" and the first for "mp3 dj software", so I think that is worth keeping. However, I don't want the title to seem like it is only for MP3s.